src/facade/facade_types.h

```diff
@@ -95,6 +95,7 @@ struct ReplyStats {
   size_t io_write_cnt = 0;
   size_t io_write_bytes = 0;
   absl::flat_hash_map<std::string, uint64_t> err_count;
+  absl::flat_hash_map<std::string, std::string> script_error_map;
```

Comment:
please add a comment saying what it maps to what

Reply:
done